active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,565 @@
+
+A teaching tip can be used without a pointer when the information that it is presenting is not specific to a UI element. 
+
+![A sample app with a teaching tip in the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipSampleApp.jpg)
+
+A teaching tip can require manual acknowledgement before dismissing or a teaching tip may also be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+A teaching tip can require the user to dismiss it via an "X" button in a top corner or a "Close" button at the bottom. A teaching tip may also be light-dismiss enabled in which case there is no dismiss button and the teaching tip will instead dismiss when a user scrolls or interacts with other elements of the application. Because of this behavior, light-dismiss tips are the best solution when a tip needs to be placed in a scrollable area. 
+
+![A sample app with a light-dismiss teaching tip in the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to."](TeachingTipLightDismissSample.jpg)
+
+
+### Create a teaching tip
+
+Here's the XAML for a pointing teaching tip control that demonstrates the default look of the TeachingTip with a title and subtitle. 
+Note that the teaching tip can appear anywhere in the element tree. In this example below, it's located in a ResourceDictionary.
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save">
+    <Button.Resources>
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Target="{x:Bind SaveButton}"
+            Title="Save automatically"
+            Subtitle="When you save your file to OneDrive, we save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </Button.Resources>
+</Button>
+```
+
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!HaveExplainedAutoSave())
+    {
+        AutoSaveTip.IsOpen = true;
+        SetHaveExplainedAutoSave();
+    }
+}
+```
+
+Here's the result when this Page containing the button and teaching tip shown:
+
+![A sample app with a teaching tip pointing at the save button. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipPointerSample.jpg)
+
+### Non-pointing tips
+
+Not all tips relate to an element onscreen. For these scenarios, do not set the Target property and the teaching tip will instead display relative to the edges of the XamlRoot. However, a teaching tip can have the pointer removed while retaining placement relative to a UI element by setting the PointerMode property to "Off". 
```

Comment:
"XamlRoot" is still difficult to understand. Maybe we should work with DMC to create a special page on the topic, refer to it more as a description with "the xaml root", and make it a hyperlink?

Reply:
Changed "XamlRoot" to "the xaml root" [here](https://github.com/Microsoft/microsoft-ui-xaml-specs/commit/2bf44703a703275f75a8ea53e1ca5ced6009ba95). I agree that a page special page on the topic may increase clarity. @MikeHillberg, anyone from DMC we can tag here?